### PR TITLE
Feature: storedValues can be array of strings

### PR DESCRIPTION
**Pull request by MBkkt**
### Scope & Purpose

Example: ["a", "b"] instead of [["a"], ["b"]]

- [ ] :hankey: Bugfix
- [ ] :pizza: New feature
- [ ] :fire: Performance improvement
- [ ] :hammer: Refactoring/simplification

### Checklist

- [ ] Tests
  - [ ] **Regression tests**
  - [ ] C++ **Unit tests**
  - [ ] **integration tests**
  - [ ] **resilience tests**
- [ ] :book: CHANGELOG entry made
- [ ] :books: documentation written (release notes, API changes, ...)
- [ ] Backports
  - [x] Backport for 3.10: *(Please link PR)*
  - [ ] Backport for 3.9: *(Please link PR)*
  - [ ] Backport for 3.8: *(Please link PR)*

#### Related Information

*(Please reference tickets / specification / other PRs etc)*

- [ ] Docs PR: 
- [ ] Enterprise PR:
- [ ] GitHub issue / Jira ticket:
- [ ] Design document:

### CODE REVIEW

File: tests/js/common/aql/aql-view-search-alias-noncluster.js [hunk 1]
```diff
@@ -92,7 +92,7 @@ jsunity.run(function IResearchAqlTestSuite_SearchAlias_NonCluster() {
                        fields:["field1", "field2", "field3"],
                        name:"invertedIndexWithLongPrimarySort",
                        primarySort: {fields:[{field: "field1", direction: "asc"}]},
-                       storedValues: [["field2"], ["field3"]]});
```
Comment (gnusi) [inline tests/js/common/aql/aql-view-search-alias-noncluster.js:95]: We still have to test both versions

**Review by gnusi — CHANGES_REQUESTED** [review]

File: tests/js/common/aql/aql-view-search-alias-noncluster.js (hunk 1 above)
Comment (MBkkt) [reply tests/js/common/aql/aql-view-search-alias-noncluster.js:95]: We have above or below

File: arangod/IResearch/AqlHelper.cpp [hunk 2]
```diff
@@ -718,25 +706,23 @@ bool nameFromAttributeAccess(
       }
     }
 
-    void append(std::string_view const& value) {
```
Comment (gnusi) [inline arangod/IResearch/AqlHelper.cpp:721]: .

**Review by Dronplane — APPROVED** [review]
LGTM

File: arangod/IResearch/AqlHelper.cpp (hunk 2 above)
Comment (MBkkt) [reply arangod/IResearch/AqlHelper.cpp:721]: Fixed

**Review by gnusi — APPROVED** [review]

**Pull request merged**